test/test_indexing.py

```diff
@@ -727,6 +729,7 @@ def test_byte_mask_accumulate(self, device):
             self.assertEqual(y, torch.ones(size=(10, 10), device=device))
             self.assertEquals(len(w), 2)
 
+    @unittest.skipIf(IS_JETSON, "Too large for Jetson")
```

Comment:
I assume this test runs OOM? If so, would it make more sense to check the memory requirements via e.g. https://github.com/pytorch/pytorch/blob/67941c8a94e1731c1f33f37915d210ec244be589/test/test_cuda.py#L57-L58

Reply:
okay will do